core/src/main/java/org/hisp/dhis/android/core/constant/internal/ConstantCoroutineCallFactory.kt

```diff
@@ -40,14 +40,14 @@ import org.koin.core.annotation.Singleton
 internal class ConstantCoroutineCallFactory(
     data: GenericCallData,
     coroutineAPICallExecutor: CoroutineAPICallExecutor,
-    private val service: ConstantService,
     private val handler: ConstantHandler,
+    private val constnatGetCall: ConstantGetCallInterface,
```

Comment:
Hey! There's a typo on `constnatGetCall` . 😄

Reply:
Maybe `ConstantProvider`, `ConstantFetcher`, `ConstantCallProvider`, `ConstantCallGetter` are better names. 
Also maybe the val property could be `callProvider`,  `callFetcher` or `callGetter`.

Reply:
Also, we try to avoid use the `Interface` suffix and add a `Impl` suffix on the implementation when there is only one.